connectors/sources/confluence.py

```diff
@@ -195,6 +215,17 @@ async def _handle_client_errors(self, url, exception):
 
             await self._sleeps.sleep(retry_seconds)
             raise ThrottledError
+        elif exception.status == 400:
+            self._logger.error(f"Bad Request Error (400). Exception: {exception}.")
+            raise UnauthorizedException
```

Comment:
Should this be "Unauthorized" here? For a 400?

Reply:
We are already raising `UnauthorizedException` for 401 updated BadRequestError for 400